src/Views/Form/Templates/Sequoia/views/receipt.php

```diff
@@ -35,15 +35,46 @@
 		<?php if ( isset( $options['thank-you']['sharing'] ) && $options['thank-you']['sharing'] === 'enabled' ) : ?>
 			<div class="social-sharing">
 				<p class="instruction">
-					<?php __( 'Tell the world about your generosity and help spread the word!', 'give' ); ?>
+					<?php echo $options['thank-you']['sharing_instruction']; ?>
 				</p>
 				<div class="btn-row">
-					<button class="give-btn social-btn facebook-btn">
+					<!-- Use inline onclick listener to avoid popup blockers -->
+					<button class="give-btn social-btn facebook-btn"
+						onclick="
+							// Retrieve and sanitize url to be shared
+							let url = parent.window.location.toString();
+							if (url.includes('?giveDonationAction=showReceipt')) {
+								url = url.replace('?giveDonationAction=showReceipt', '');
+							}
+							// Calculate new window position, based on parent window height/width
+							const top = parent.window.innerHeight / 2 - 365;
+							const left = parent.window.innerWidth / 2 - 280;
+							// Open new window with prompt for Facebook sharing
+							window.open(`https://www.facebook.com/sharer/sharer.php?u=${url}`, 
+							'newwindow', 
+							`width=560,height=730,top=${top},left=${left}`); 
+							return false;">
 						<?php _e( 'Share on Facebook', 'give' ); ?><i class="fab fa-facebook"></i>
 					</button>
-					<button class="give-btn social-btn twitter-btn">
+					<!-- Use inline onclick listener to avoid popup blockers -->
+					<button
+						class="give-btn social-btn twitter-btn"
+						onclick="
+							// Retrieve and sanitize url to be shared
+							let url = parent.window.location.toString();
+							if (url.includes('?giveDonationAction=showReceipt')) {
```

Comment:
@henryholtgeerts use `Give.fn.getParameterByName` to get data by param key

Reply:
Done, ref: https://github.com/impress-org/givewp/commit/1a56a5bf2e33f0ca14064c39fa4021d03590864f